server/graphql/v2/object/OrderPermissions.ts

```diff
@@ -23,7 +23,7 @@ export const canMarkAsExpired = async (req: express.Request, order): Promise<boo
 };
 
 export const canEdit = async (req: express.Request, order): Promise<boolean> => {
-  return order.status === ORDER_STATUS.PENDING && isHostAdmin(req, order);
+  return order.status === ORDER_STATUS.PENDING && isHostAdmin(req, order) && order.data?.isPendingContribution;
```

Comment:
`isHostAdmin` returns a promise, we must await if not returning it

```suggestion
  return order.status === ORDER_STATUS.PENDING && await isHostAdmin(req, order) && order.data?.isPendingContribution;
```